commands/spublish.md

```diff
@@ -8,7 +8,7 @@ For more information about sharded pubsub, see [Sharded Pubsub](/topics/pubsub#s
 
 @return
 
-@integer-reply: the number of clients that received the message.
+@integer-reply: the number of clients that received the message. Note that in a Redis Cluster, only clients that are connected to the same node as the publishing client are included in the count.
```

Comment:
```suggestion
@integer-reply: the number of clients that received the message.
Note that in a Redis Cluster, only clients that are connected to the same node as the publishing client are included in the count.
```

Reply:
It's just a copy from PUBLISH, should I change it there too?